config.go

```diff
@@ -23,6 +23,15 @@ type Loader struct {
 	Tag string
 }
 
+// SliceParser parses a raw input string as a slice of strings accordingly to a built-in slice separator
+type SliceParser func(src string) []string
```

Comment:
I'm not sure that it's worth having a type for this, as the function type is so simple.
Also, I don't think I'd make a global mutable variable to hold it - it would be nicer, I think, to make it something that could be configured on a given Loader, much like we can set configuration on a json.Decoder.

How about something like this?

     // SetSliceSplitter sets the function that is used to split strings before
     // assigning them to a slice value. By default strings.Split(s, ",") is used.
     func (l *Loader) SetSliceSplitter(f func(s string) []string)